app/views.py

```diff
@@ -33,3 +48,11 @@ def page_forbidden(error):
 @url.errorhandler(500)
 def page_server_error(error):
     return render_template('500.html', title='Server Error'), 500
+
+
+def convert_to_readable_time(time):
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
```

Comment:
Different countries have different standards (e.g. In the Philippines, today is "5/8/2016 "but in Australia it is "8/5/2016") so it is best to stick with a format that anyone can understand.

Try `%F %I:%M:%S %p`. It would give you `2016-04-08 12:03:19 PM`.